package.json

```diff
@@ -28,5 +28,8 @@
     "husky": "4.2.5",
     "semantic-release": "17.0.7",
     "semantic-release-slack-bot": "1.6.2"
+  },
+  "resolutions": {
+    "**/lodash": "4.17.19"
```

Comment:
Really not a fan of this, I don't understand how we can confidently bypass dependencies rules.
This feature should be used only in case of "emergency" in my opinion.

Reply:
Here is my update proposal to get rid of the lodash vulnerability:
https://github.com/ForestAdmin/forest-rails/pull/372

Reply:
I went for the "quickest" solution but your approach is by far the cleanest. Let's go for it.